LIS3DH.py

```diff
@@ -137,7 +143,7 @@ def getAxis(self, axis):
       if range==self.RANGE_2G:    divisor = 16380
       elif range==self.RANGE_4G:  divisor = 8190
       elif range==self.RANGE_8G:  divisor = 4096
-      elif range==self.RANGE_16G: divisor = 2048
+      elif range==self.RANGE_16G: divisor = 1365.33
```

Comment:
Where did you get this number from? I was going off [this](https://github.com/adafruit/Adafruit_LIS3DH/blob/master/Adafruit_LIS3DH.cpp#L159) which seems to indicate that 2048 is the correct divisor for 16G?